updated to reflect new diagnostics frequency (#3601)
mattermost/docs: source/administration/telemetry.rst

.. note:: 
Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).

> Deployment and Server Configuration snapshots

Reviewer: ```suggestion
Deployment and server configuration snapshots
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Reporting Frequency

Reviewer: ```suggestion
Reporting frequency
```

>   - Upon booting up the server for the first time: every 10 minutes for the first hour, then every hour for the first 12 hours 

Reviewer: ```suggestion
  - When booting up the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.
```

Reviewer: Does this apply after a reboot too, or just on the very first boot?

>   - At the 24 hour mark and every 24 hours 

Reviewer: ```suggestion
  - At the 24 hour mark and every 24 hours thereafter.
```

> Deployment Configuration Information

Reviewer: ```suggestion
Deployment configuration information
```

>   Basic information including Mattermost server version, database and operating system type and version, and count of System Administrator accounts

Reviewer: ```suggestion
  Basic information including Mattermost server version, database and operating system type and version, and count of System Administrator accounts.
```

.. note:: 
Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).

> Deployment and Server Configuration snapshots

Reviewer 2: ```suggestion
Deployment and Server Configuration Data
```

Author: yep better thx

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reporting Frequency

>   - When booting up the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.

Reviewer 2: ```suggestion
  - When starting the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.
```

Reviewer 2: "booting up" translates to "start-up" in Japanese - so suggesting the use of "starting" for simplicity

Author: ok

.. note:: 
Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).

> Reporting Frequency

Reviewer 2: Big +1 for being very specific about the frequency.

However, one suggestion: Mattermost has customers around the world and in many cases English is their second language. Few of the words and phrases used here have multiple meanings and may result in misunderstood terms.

One example: If I take `Every hour thereafter for the first 12 hours` and use [Google Translate](https://www.google.com/search?q=english+to+japanese&rlz=1C5CHFA_enCA768CA768&oq=english&aqs=chrome.1.69i57j69i59j0l4j69i59j0.1744j0j7&sourceid=chrome&ie=UTF-8) to translate it to Japanese, then back to English, I get `For the first 12 hours thereafter, every hour`, which is not quite the same information you intended to convey (but is what a non-native language speaking individual would get if they used a translator)

Thus, propose revisiting so that a non-English native speaker with basic computer skills can easily understand it, raising the quality of the section.

  - Every 10 minutes for the first hour after ther server is booted up for the first time
  - Every hour thereafter for the first 12 hours
  - At the 24 hour mark and every 24 hours after

> The following data is sent:

Reviewer 2: Currently, there are two subsections for "Error and Diagnostics Reporting Feature" 
 - [The following data is sent once every 24 hours](https://docs.mattermost.com/administration/telemetry.html#the-following-data-is-sent-once-every-24-hours)
 - [The following information is sent when the specified event occurs](https://docs.mattermost.com/administration/telemetry.html#the-following-information-is-sent-when-the-specified-event-occurs)

Does the new reporting frequency apply to the first of these subsections only? If so, let's make that clear

Author: OK made a couple more edits. Hopefully clearer now, but had to re-title some sections for which there isn't a clear title. Let me know if it works better

> Aggregated Usage Statistics

Reviewer: ```suggestion
Aggregated usage statistics
```

  Non-personally identifiable summations of basic usage statistics: Number of enabled and disabled accounts, number of user logins in the last 24 hours and the last 30 days, number of users active in the last day/month, whether APIv3 endpoints were used in the last 24 hours, number of posts, channels, teams, and bots.

Event data
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Reporting Frequency

Reviewer: ```suggestion
Reporting frequency
```

>   - Immediately after the specific event occurs

Reviewer: ```suggestion
  - Immediately after the specific event occurs.
```

  .. note::

>     Majority of these events have been disabled since Mattermost v5.8. Refer to the source file for the `current list of events sent via telemetry <https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L3069>`_.

Reviewer: ```suggestion
The majority of these events have been disabled since Mattermost v5.8. Refer to the source file for the `current list of events sent via telemetry <https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L3069>`_.
```
